lib/helpers/function.ts

```diff
@@ -0,0 +1,58 @@
+export const noop = () => {};
+export const identity = (value: any) => value;
+
+export const debounce = (func: Function, wait: number, isImmediate?: boolean) => {
```

Comment:
on debounced function types would become `(...any[]) => void` which is not helpful.

```suggestion
export const debounce = <A extends any[]>(func: (...args: A[]) => void, wait: number, isImmediate?: boolean) => {
```